libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFSecurityLockout.m

```diff
@@ -219,16 +219,6 @@ + (BOOL)needsPasscodeFlow:(NSUInteger)newLockoutTime {
     return newLockoutTime !=0 || [self usersHavePasscodePolicy];
 }
 
-+ (BOOL)needsPasscodePolicyChange:(NSUInteger)newLockoutTime passcodeLength:(NSUInteger)newPasscodeLength {
-    BOOL result = NO;
-    if (securityLockoutTime == 0 && newLockoutTime > 0) {
-        result = YES;
-    }else if (newLockoutTime !=0 && newLockoutTime < securityLockoutTime){
-         result = YES;
-    }
-    return result || (newPasscodeLength > [self passcodeLength]);
```

Comment:
This line is dangerous due to the or.  Since later on we just check != instead of <, it was possible to increase the lockout time if the same policy also increased the passcode length.

Reply:
@brandonpage Let's discuss this offline.